elixir/apps/web/lib/web/live/actors/edit.ex

```diff
@@ -17,7 +17,7 @@ defmodule Web.Actors.Edit do
           actor: actor,
           groups: groups,
           form: to_form(changeset),
-          page_title: "Edit actor #{actor.name}"
```

Comment:
I think it's better to have an explicit title for edit/new/etc pages, imagine you try to edit multiple users at once - you open a few tabs, and they all will be named "Actors" while if we keep the old title you can tell which actor is there by the browser tab name.

Reply:
Happy to discuss this more, but I feel like if that's the case, the likelihood that you'd be able to read the actual browser tab titles is pretty low.  If you're used to opening multiple tabs for Firezone, you're probably opening a lot of other tabs as well, which means the tab titles get to be pretty small and you wouldn't be able to see the actor name.

All that said, I'm not hard against this, just had a gut feeling it wouldn't add too much value.
@jamilbk, do you have a preference on this?

Reply:
Hmm no super strong preference, but I do lean towards keeping them action-scoped at least. That if I'm working to change a lot of actors/sites/etc (and I don't or can't use the REST API) I can open them in tabs and at least have some chance of seeing which tabs are which.

How about this?

- show view: `#{record.name || "Site #{record.id}"} • Firezone`
- edit view: `Edit #{record.name} • Firezone`
- index view: `Sites • Firezone`
- new view: `New Site • Firezone`

Reply:
Ok, sounds good.  I must be in the minority of not really looking at my browser tab titles 😄 
I'll get the PR updated